packages/qdrant-loader/tests/unit/connectors/jira/test_jira_config.py

```
@@ -263,3 +263,219 @@ def test_dc_multiple_placeholders_all_reported(self):
+        assert cfg.updated_after.day == 4
+
+    # ── Relative date formats ──────────────────────────────────────────────────
+
+    def test_relative_days_format(self):
+        """Relative format '-2 days' should be parsed."""
+        from datetime import datetime, timedelta
+
+        cfg = _make(updated_after="-2 days")
+        assert isinstance(cfg.updated_after, datetime)
+        # Check that it's approximately 2 days ago (allow 1 second tolerance)
+        expected = datetime.now() - timedelta(days=2)
```

Ruff (0.15.14)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)